modules/heat_conduction/tutorials/introduction/compare_solutions.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```

Comment:
sure you don't want to name this `therm_cmpr_sols.py` ?

Reply:
Actually, I was just changing the name to something like that.